bin/thoth

```diff
@@ -1,45 +1,45 @@
-#!/usr/bin/env node
+#!/usr/bin/env node --abort-on-uncaught-exception
 
 /*
  * Copyright 2020 Joyent, Inc.
  */
 
+var mod_assert = require('assert');
+var mod_bunyan = require('bunyan');
+var mod_child = require('child_process');
+var mod_crypto = require('crypto');
 var mod_ctype = require('ctype');
-var mod_path = require('path');
 var mod_fs = require('fs');
-var util = require('util');
-var mod_crypto = require('crypto');
-var mod_bunyan = require('bunyan');
+var mod_jsprim = require('jsprim');
 var mod_manta = require('manta');
-var mod_zlib = require('zlib');
-var mod_child = require('child_process');
+var mod_path = require('path');
 var mod_r = require('rethinkdb');
+var mod_stream = require('readable-stream');
+var mod_tmp = require('tmp');
+var mod_vasync = require('vasync');
 var mod_vm = require('vm');
-var mod_assert = require('assert');
-var mod_jsprim = require('jsprim');
-
-var mod_stream = require('stream');
-if (!mod_stream.Readable) {
-	/*
-	 * If we're on node 0.8, pull in streams2 explicitly.
-	 */
-	mod_stream = require('readable-stream');
-}
 
+var util = require('util');
 var Readable = require('stream').Readable;
 var sprintf = require('sprintf').sprintf;
 var endsWith = mod_jsprim.endsWith;
+var Int64 = require('node-int64');
+
 var ThothListStream = require('../lib/liststream').ThothListStream;
 var forEachStream = require('../lib/foreachstream');
 var batchStream = require('../lib/batchstream');
 var mapStream = require('../lib/mapstream');
 var jsondiff = require('../lib/jsondiff');
+var vasync_extra = require('../lib/vasync-extra');
+
+var thothVersion = require('../package').version;
```

Comment:
nit: I'd suggest explicitly saying `require('../package.json')` with the extension there. No need to have it also look for package.js.